### ISSUE TITLE
loss问题

### ISSUE THREAD

**Issue by znlinux** [CCIIPLab/GCE-GNN#9]
作者您好，
loss = model.loss_function(scores, targets - 1) 
代码中loss部分scores是预测结果？（我没有发现softmax层），targets-1的意图是什么呢？
